shadows/framework/src/main/java/org/robolectric/shadows/ShadowBiometricManager.java

```diff
@@ -65,9 +109,37 @@ public void setCanAuthenticate(boolean flag) {
     biometricServiceConnected = flag;
   }
 
-  @Implementation(minSdk = R)
-  protected int canAuthenticate(int userId, int authenticators) {
-    return biometricServiceConnected ? BIOMETRIC_SUCCESS : BIOMETRIC_ERROR_NO_HARDWARE;
+  /**
+   * @param num to set the number of enrolled biometric. If sets the value equal or below 0, result
+   *     will be {@link BiometricManager#BIOMETRIC_ERROR_NONE_ENROLLED}
+   */
+  public void setEnrolledBiometrics(int num) {
+    enrolledBiometrics = num;
+  }
+
+  /**
+   * @param flag to set has biometric hardware or not. If sets the value false, result will be
+   *     {@link BiometricManager#BIOMETRIC_ERROR_HW_UNAVAILABLE}
+   */
+  public void hasAvailableHardware(boolean flag) {
```

Comment:
IMO, the name `hasAvailableHardware` is used for getter, instead of setter. What about `setHardwareAvailable`?